CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- **CUMULUS-1958**
+  - Updated deploy process to utilize a bootstrap 'tea-map-cache' lambda that will, after deployment of Cumulus Core's TEA instance, query TEA for all protected/public buckets and generate a mapping configuration used internally by Core.  This object is also exposed as an output of the Cumulus module as `distribution_bucket_map`.
```

Comment:
Do we need more details here, or will there be updates to docs elsewhere? In particular, I'm looking for an explanation of what the "mapping configuration" is. Mapping what to what, and why?

Reply:
Ah - sure.    So the userland doc updates are in https://github.com/nasa/cumulus/pull/1698

Basically a bucket->tea path map .json file we're using internally and exposing via `cumulus` and `distribution` module outputs.     

This object prevents us from having to have all CMR distribution link related calls hit TEA's endpoint *per file*.